esrally/mechanic/telemetry.py

```diff
@@ -142,22 +143,33 @@ def instrument_env(self, car, candidate_id):
         time.sleep(3)
 
         console.info("%s: Writing flight recording to [%s]" % (self.human_name, log_file), logger=logger)
-        # this is more robust in case we want to use custom settings
-        # see http://stackoverflow.com/questions/34882035/how-to-record-allocations-with-jfr-on-command-line
-        #
-        # in that case change to:
-        #
-        # Java 8: -XX:StartFlightRecording=defaultrecording=true,settings=es-memory-profiling
-        # Java 9+: "-XX:StartFlightRecording=maxsize=0,maxage=0s,disk=true,dumponexit=true,settings=es-memory-profiling,filename=%s"
+
+        java_opts = self.java_opts(log_file)
+
+        logger.info("jfr: Adding JVM arguments: [%s].", java_opts)
```

Comment:
Could use the new .format() here.

Reply:
Ah silly me. It's the logger. Ignore.

Reply:
I think this is a different case here and I wanted to discuss this separately anyway. It looks as if we are using old style string formatting but we are actually not. This is the standard way the [Python logging](https://docs.python.org/3/library/logging.html) facility is used and I think it would make sense that we go with the default here.

Reply:
Ok, I'll merge then. :)